packages/manager/apps/catalog-revamp/src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,18 @@
+.searchbar-container {
+  margin: 20px;
+  float: right;
+
+  .searchbar {
+    float: right;
+
+    .search-button {
+      position: absolute;
+      margin-left: -10px;
+    }
+
+    osds-input {
+      min-width: 250px;
+      display: inline-block;
+    }
+  }
+}
```

Comment:
**nitpick:** Avoid custom CSS and use tailwind. Also, prefer using flexbox/grid over `float`

Reply:
Done, tailwind added ✅